ert3/config/_ensemble_config.py

```diff
@@ -31,12 +32,12 @@ class Input(_EnsembleConfig):
 
 class EnsembleConfig(_EnsembleConfig):
     forward_model: ForwardModel
-    input: Tuple[Input, ...]
+    inputs: Tuple[Input, ...]
```

Comment:
I would rather use singular, as it reads a bit better. The input can be used as uncountable while it might contain multiple items, "I did not provide input to the following discussion" I also see input to an ensemble as *a* list of records

Reply:
I can see how the singular form 'input' and 'output' makes sense when used in natural language such as English. And I do agree with context, such as the one you gave, one or more topics/subject is implicit. but when read with no context (by itself) to label an object the word is consumed more explicitly and concisely and should tell it correctly with no convolution.